src/components/pages/article/article-page.tsx

```diff
@@ -31,18 +32,19 @@ export type ArticlePageProps = {
 
 export const ArticlePage = (props: ArticlePageProps): JSX.Element => {
   const [activeTab, setActiveTab] = useState<string>(props.activeTab);
+  const id = props.msidWithVersion ?? props.metaData.msid;
```

Comment:
Not requiring `msidWithVersion` was really an approach that I suspected would allow for less refactoring on the tests and we should probably make it a required property and refactor the tests.

`msidWithVersion` in it's current implementation can be one of 3 variations:

- [msid] (for published latest)
- [msid]v[version] (for published version specific)
- [msid]/v[version] (for previews)

These have more to do with the path and so `msidWithVersion` might not be the best name. We could call it `routeId`. I will refactor this and drop the `props.metaData.msid` fallback and then we will deal with the refactors to the tests.

Reply:
Yes, I think we're begging to be more explicit with these IDs. Proposal:
msid: 12345
doi: 10.7554/12345
versionId(entifier): 2
versionDoi: 10.7554/12345.2
routeId: 12345v2
contentId: 12345/v2

Reply:
Let's create a ticket with the above

Reply:
Created: https://github.com/elifesciences/enhanced-preprints-issues/issues/621